flask_monitoringdashboard/views/deployment.py

```diff
@@ -45,3 +44,33 @@ def deploy_config():
             'colors': config.colors,
         }
     )
+
+
+@blueprint.route('/api/deploy_alert_config')
+@secure
+def deploy_alert_config():
+    """
+    :return: A JSON-object with alert configuration details
+    """
+    # TODO
+    # post_to_back_if_telemetry_enabled(**{'name': 'deploy_alert_config'})
```

Comment:
what is this for? not for sending telemetry info about the installation is it? because then email is too private to collect it.